.circleci/config.yml

```diff
@@ -366,10 +473,60 @@ workflows:
           name: e2e-test ExecuteCleanup
           test_name: ExecuteCleanup
           requires:
-            - unit-test
+            - unit-test-casskop
           filters:
             tags:
               only: /.*/
             branches:
               # Forked pull requests have CIRCLE_BRANCH set to pull/XXX
               ignore: /pull\/[0-9]+/
+
+  # CassKop operator workflow.
+  ci-multicasskop-operator:
+    jobs:
+      - operator/generate:
```

Comment:
@erdrix, those (generate) should not be jobs but steps

Reply:
But something is not clear to me, how can we publish a docker image on the docker hub, before having played the unit-tests ?To me, it doesn't make sense, but maybe I'm missing something ?

Reply:
It should be published but only as a temporary image or branch release to be used by all tests. Then only when the PR is merged the release (*-master) should be published

Reply:
We should also have a job to publish a release when we tag it. Probably need a card for that ?